crates/iota-sdk-types/src/transaction/serialization.rs

```diff
@@ -772,8 +772,7 @@ mod signed_transaction {
     ///
     /// enum AppId {
     ///     Iota = 0,
-    ///     Narwhal = 1,
-    ///     Consensus = 2,
+    ///     Consensus = 1,
     /// }
     ///
     /// enum IntentScope {
```

Comment:
Please put these sections in the same order as in the Intent: scope, version, id 🙏🏻